### ISSUE TITLE
notfify.magicmirror

### ISSUE THREAD

**Issue by fribse** [sindrebroch/ha-magicmirror#11]
## Version of the custom_component

1.3.0
## Configuration

Single MagicMirror connected on IP

## Describe the bug
When I try to send a notification, I just get an error about calling the service.
I call the service through the Developer tools interface

```
service: notify.magicmirror
data:
  message: 'Det ringer på døren'

```

## Debug log

```text
Denne fejl stammer fra en brugerdefineret integration.

Logger: homeassistant.helpers.script.websocket_api_script
Source: custom_components/magicmirror/notify.py:58
Integration: Magic Mirror (documentation, issues)
First occurred: 14.51.39 (4 occurrences)
Last logged: 15.16.54

websocket_api script: Error executing script. Unexpected error for call_service at pos 1: alert() got an unexpected keyword argument 'message'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 447, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 680, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1704, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1741, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/usr/src/homeassistant/homeassistant/components/notify/legacy.py", line 229, in _async_notify_message_service
    await self.async_send_message(**kwargs)
  File "/config/custom_components/magicmirror/notify.py", line 58, in async_send_message
    await self._notify.alert(
TypeError: alert() got an unexpected keyword argument 'message'
```

**Comment by sindrebroch** [comment]
Thanks for the issue. I am able to reproduce, will look into it later.

Been a while since I used the notify, so not sure if it's recent or not.

**Comment by sindrebroch** [comment]
Should be fixed in release 1.4.0. Please reopen if the issue persists.
